sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/tests/ServerTrustGroupTests.cs

```diff
@@ -0,0 +1,153 @@
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.Management.ResourceManager.Models;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using System.Collections.Generic;
+using Xunit;
+using Microsoft.Azure.Management.Network;
+using Sku = Microsoft.Azure.Management.Sql.Models.Sku;
+using Microsoft.Azure.Management.Network.Models;
+using Microsoft.Rest.Azure;
+
+namespace Sql.Tests
+{
+	/// <summary>
+	/// Contains tests for the lifecycle of a sanaged instance and managed database security alert policy
+	/// </summary>
+	public class ServerTrustGroupsTest
+	{
+		private void createManagedInstances(SqlManagementTestContext context, ResourceGroup resourceGroup, IList<string> managedInstanceNames)
+		{
+			SqlManagementClient sqlClient = context.GetClient<SqlManagementClient>();
+
+			Sku sku = new Sku();
+			sku.Name = "MIGP8G4";
+			sku.Tier = "GeneralPurpose";
+
+			VirtualNetwork vnet = ManagedInstanceTestFixture.CreateVirtualNetwork(context, resourceGroup, TestEnvironmentUtilities.DefaultLocationId);
+
+			foreach(string miName in managedInstanceNames) 
+			{
+				ManagedInstance managedInstance = sqlClient.ManagedInstances.CreateOrUpdate(resourceGroup.Name, miName, new ManagedInstance()
+				{
+					AdministratorLogin = SqlManagementTestUtilities.DefaultLogin,
+					AdministratorLoginPassword = SqlManagementTestUtilities.DefaultPassword,
+					SubnetId = vnet.Subnets[0].Id,
+					Tags = new Dictionary<string, string>(),
+					Location = TestEnvironmentUtilities.DefaultLocationId,
+					Sku = sku,
+				});
+				Assert.NotNull(managedInstance);
+			}
+		}
+
+		private IList<ServerInfo> createGroupMembers(SqlManagementClient sqlClient, string resourceGroupName, IList<string> miNames)
+		{
+			IList<ServerInfo> groupMembers = new List<ServerInfo>();
+
+			foreach(string miName in miNames)
+			{
+				groupMembers.Add(new ServerInfo($"/subscriptions/{sqlClient.SubscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{miName}"));
+			}
+
+			return groupMembers;
+		}
+
+		private void deleteManagedInstances(SqlManagementClient sqlClient, string resourceGroupName, IList<string> miNames)
+		{
+			foreach(string miName in miNames)
+			{
+				sqlClient.ManagedInstances.Delete(resourceGroupName, miName);
+			}
+		}
+
+		[Fact]
+		public void TestCreateGetDropServerTrustGroup()
+		{
+			string stgName = "stg-test";
+			IList<string> managedInstanceNames = new List<string>() { "mi1-test", "mi2-test" };
+			using (SqlManagementTestContext context = new SqlManagementTestContext(this))
+			{
+				SqlManagementClient sqlClient = context.GetClient<SqlManagementClient>();
+
+				ResourceGroup resourceGroup = context.CreateResourceGroup(TestEnvironmentUtilities.DefaultLocationId);
+				Assert.NotNull(resourceGroup);
+				createManagedInstances(context, resourceGroup, managedInstanceNames);
```

Comment:
Consider putting this in a separate Setup method, as it's used in all tests cases in the same manner. Same goes for deleting managed instances - Cleanup method.